modules/sdk/src/main/scala/org/tessellation/sdk/package.scala

```diff
@@ -47,4 +55,31 @@ package object sdk {
       EventTrigger.getClass -> 530,
       TimeTrigger.getClass -> 531
     ).union(sharedKryoRegistrar)
+
+  private val mainnetPriorityPeerIds = Set(
+    "e0c1ee6ec43510f0e16d2969a7a7c074a5c8cdb477c074fe9c32a9aad8cbc8ff1dff60bb81923e0db437d2686a9b65b86c403e6a21fa32b6acc4e61be4d70925",
+    "710b3dc521b805aea7a798d61f5d4dae39601124f1f34fac9738a78047adeff60931ba522250226b87a2194d3b7d39da8d2cbffa35d6502c70f1a7e97132a4b0",
+    "629880a5b8d4cc6d12aec26f24230a463825c429723153aeaff29475b29e39d2406af0f8b034ba7798ae598dbd5f513d642bcbbeef088290abeadac61a0445d6"
+  ).map(s => PeerId(Hex(s))).toSortedSet
+
+  // TODO populate sets with correct values
+  private val devPriorityPeerIds = Set(
+    "0174825ea2f952d7795475b690ffcc9960c486e7809fa9973e9db42cfc4f6360890d80a0e9a2363a5acf6c04cc188fa6405fae4bfdb9055b7361e0bbfe78941c"
+  ).map(s => PeerId(Hex(s))).toSortedSet
+
+  private val testnetPriorityPeerIds = Set(
+    "0174825ea2f952d7795475b690ffcc9960c486e7809fa9973e9db42cfc4f6360890d80a0e9a2363a5acf6c04cc188fa6405fae4bfdb9055b7361e0bbfe78941c"
+  ).map(s => PeerId(Hex(s))).toSortedSet
+
+  private val integrationnetPriorityPeerIds = Set(
+    "0174825ea2f952d7795475b690ffcc9960c486e7809fa9973e9db42cfc4f6360890d80a0e9a2363a5acf6c04cc188fa6405fae4bfdb9055b7361e0bbfe78941c"
+  ).map(s => PeerId(Hex(s))).toSortedSet
```

Comment:
I think it's really really bad idea to hardcode these peer ids inside the codebase. It creates weird circular reference and locks these peer ids basically. We would not be able to quickly change them (and also, you don't know peer ids for dev environment). What do you think about propagating them from the genesis node when starting the network? In the same way we keep the seedlist aligned inside the network (layer).

You may require the genesis or initial node to provide priority peers (fail otherwise) and those peers would be propagated during joining process to other peers as "base" priority peers. If there are none specified by the nodes, they can fallback to the ones propagated within the network. It would work in the same way as hardcoded here in the codebase, but first of all, we don't hardcode them (no strict dependency in the code) and secondly, we have no issue of running another environment or dev environments just like that. 

cc: @kpudlik @TheMMaciek @ryle-goehausen

Reply:
@marcinwadon This is what was decided at the last meeting. We can discuss it again at the next one.